src/test/java/org/prgrms/memory/CustomerDBMemoryTest.java

```diff
@@ -0,0 +1,155 @@
+package org.prgrms.memory;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.NoSuchElementException;
+import org.prgrms.customer.Customer;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DataAccessException;
+import org.springframework.test.context.junit.jupiter.SpringJUnitConfig;
+
+@SpringJUnitConfig
+@TestInstance(Lifecycle.PER_CLASS)
```

Comment:
```suggestion
@SpringJUnitConfig
@TestInstance(Lifecycle.PER_CLASS)
```

이 두 annotiaion은 처음보는데, 어떤 역할을 하는걸까요?

Reply:
앗 jdbcBase 에설정해줬어야 했는데 제가 잘못 달아놨네요 
@SpringJUnitConfig 는 Test class 내부에 자체적으로 Configuration을 설정하고 이를 자동 주입받을 때 사용하는 어노테이션이구, @TestInstance 는 모든 테스트 메서드를 동일한 인스턴스 환경에서 동작시키고 싶을 때 사용한다고 합니다..!